modules/social_features/social_user/social_user.install

```diff
@@ -228,6 +226,15 @@ function social_user_update_8810() {
   ]);
 }
 
+/**
+ * Revoke the permission select account cancellation method.
+ */
+function social_user_update_8811(): void {
+  user_role_revoke_permissions(RoleInterface::AUTHENTICATED_ID, [
+    'select account cancellation method',
+  ]);
+}
```

Comment:
Why duplicated update hook? If the intention here is to be backward compatible, then you should create a separate PR to 11.0.x and 10.3.x with the correct hook number.